lib/codecs/h264.js

```diff
@@ -61,7 +61,9 @@ NalByteStream = function() {
         }
 
         // deliver the NAL unit
-        this.trigger('data', buffer.subarray(syncPoint + 3, i - 2));
+        if (buffer.subarray(syncPoint + 3, i - 2).length > 0) {
```

Comment:
Don't create a subarray just to check the length when you can just make sure that `syncPoint + 3 !== i - 2`!